deploy/kubernetes/Makefile

```diff
@@ -19,21 +20,21 @@ test: generate-k8s
 clean:
 	@for f in $(ALL); do rm -f "$$f-kubernetes.yaml"; done
 
-$(ALL): 
+$(ALL):
 ifdef WITHOUTCONFIG
 	@echo "Generating $@-kubernetes-without-configmap.yaml"
 	@rm -f $@-kubernetes-without-configmap.yaml
 	@for f in $(shell ls $@/*.yaml | grep -v daemonset-configmap); do \
-		sed "s/%VERSION%/VERSION/g" $$f >> $@-kubernetes-without-configmap.yaml; \
+		sed -e "s/%VERSION%/VERSION/g" -e "s/%BRANCH%/${BRANCH_VERSION}/g" $$f >> $@-kubernetes-without-configmap.yaml; \
```

Comment:
Dont change this one, because for the kibana PR we just remove the %

Word Version must remain

Reply:
In the first part the `"s/%VERSION%/VERSION/g"` is preserved so I expect it would work. I'm not sure how we could handle the second part.